src/operator/operatorconfig/config.go

```diff
@@ -1,20 +1,92 @@
 package operatorconfig
 
 import (
+	"fmt"
+	"github.com/otterize/intents-operator/src/shared"
+	"github.com/spf13/pflag"
 	"github.com/spf13/viper"
 	"strings"
 )
 
 const (
-	TelemetryErrorsAPIKeyKey     = "telemetry-errors-api-key"
-	TelemetryErrorsAPIKeyDefault = "20b1b74678347375fedfdba65171acb2"
+	TelemetryErrorsAPIKeyKey                 = "telemetry-errors-api-key"
+	TelemetryErrorsAPIKeyDefault             = "20b1b74678347375fedfdba65171acb2"
+	AWSRolesAnywhereTrustAnchorARNKey        = "rolesanywhere-trust-anchor-arn"
+	AWSRolesAnywhereSPIFFETrustDomainKey     = "rolesanywhere-spiffe-trust-domain"
+	AWSRolesAnywhereClusterName              = "rolesanywhere-cluster-name"
+	EnableAWSServiceAccountManagementKey     = "enable-aws-serviceaccount-management"
+	EnableAWSServiceAccountManagementDefault = false
+	EnableAWSRolesAnywhereKey                = "enable-aws-iam-rolesanywhere"
+	EnableAWSRolesAnywhereDefault            = false
+	MetricsAddrKey                           = "metrics-bind-address"
+	MetricsAddrDefault                       = ":7071"
+	ProbeAddrKey                             = "health-probe-bind-address"
+	ProbeAddrDefault                         = ":7072"
+	SpireServerAddrKey                       = "spire-server-address"
+	SpireServerAddrDefault                   = "spire-server.spire:8081"
+	CertProviderKey                          = "certificate-provider"
+	CertProviderDefault                      = CertProviderNone
+	CertManagerIssuerKey                     = "cert-manager-issuer"
+	CertManagerIssuerDefault                 = "ca-issuer"
+	SelfSignedCertKey                        = "self-signed-cert"
+	SelfSignedCertDefault                    = true
+	CertManagerUseClustierIssuerKey          = "cert-manager-use-cluster-issuer"
+	CertManagerUseClusterIssuerDefault       = false
+	UseCertManagerApproverKey                = "cert-manager-approve-requests"
+	UseCertManagerApproverDefault            = false
+	AWSUseSoftDeleteStrategyKey              = "aws-use-soft-delete"
+	AWSUseSoftDeleteStrategyDefault          = false
+	DebugKey                                 = "debug"
+	DebugDefault                             = false
+	EnableLeaderElectionKey                  = "leader-elect"
+	EnableLeaderElectionDefault              = false
+	EnvPrefix                                = "OTTERIZE"
+)
 
-	EnvPrefix = "OTTERIZE"
+const (
+	CertProviderSPIRE       = "spire"
+	CertProviderCloud       = "otterize-cloud"
+	CertProviderCertManager = "cert-manager"
+	CertProviderNone        = "none"
 )
 
 func init() {
+	viper.SetDefault(EnableAWSServiceAccountManagementKey, EnableAWSServiceAccountManagementDefault)
```

Comment:
Moved CLI flags to environment variables, but kept CLI flags for backwards compatibility. Haven't changed helm chart.